page_test.go

```diff
@@ -894,6 +896,53 @@ func TestPageElementFromObjectErr(t *testing.T) {
 	g.Err(p.ElementFromObject(obj.Object))
 }
 
+func TestPageTriggerFavicon(t *testing.T) {
+	g := setup(t)
+
+	// test browser in no-headless mode with an error
+	{
+		path, _ := launcher.LookPath()
+		var u *launcher.Launcher
```

Comment:
Why do you use `u` here?

```suggestion
		var l *launcher.Launcher
```